api_utils/txtlib.py

```
@@ -0,0 +1,131 @@
+            'name': f"قالب:{name}",
+            'namestrip': namestrip,
+            'params': params,
+            'item': pa_item,
+        }
+        # ---
+        result.append(ficrt)
+    # ---
+    return result
+
+
+def get_one_temp_params(text, tempname="", templates=[], lowers=False, get_all_temps=False):
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)